sklearn/decomposition/dict_learning.py

```diff
@@ -571,12 +577,19 @@ def dict_learning_online(X, n_components=2, alpha=1, n_iter=100,
 
         (U^*, V^*) = argmin 0.5 || X - U V ||_2^2 + alpha * || U ||_1
                      (U,V)
-                     with || V_k ||_2 = 1 for all  0 <= k < n_components
+                     with || V_k ||_2 <= 1 for all  0 <= k < n_components
 
     where V is the dictionary and U is the sparse code. This is
```

Comment:
the U, V here is also inconsistent with D/C as used above, and with D/alpha as used in the paper... but consistent with other decompositions algos in sklearn. Maybe we should use U/V in the other docstrings and maybe in code too?